web/pdf_page_view.js

```diff
@@ -538,22 +538,24 @@ var PDFPageView = (function PDFPageViewClosure() {
       var viewport = pdfPage.getViewport(1);
       // Use the same hack we use for high dpi displays for printing to get
       // better output until bug 811002 is fixed in FF.
-      var PRINT_OUTPUT_SCALE = 2;
+      var DPI = 200; // increase to improve quality
+      var PRINT_OUTPUT_SCALE = DPI/72; 
```

Comment:
This would significantly increase the size of the printing canvases, and thus also the memory comsumption during printing.
As is, this patch increases canvas sizes, and thus memory usage, by a factor of almost `2` or `1,93` to be exact.
Given the above, I'm not sure if we want to change the default in such a way!?